src/containers/Notifications/Notifications.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Snackbar from 'material-ui/Snackbar';
+import { hideNotification as hideNotificationActionCreator } from './actions';
+
+class Notifications extends Component {
+  constructor(props) {
+    super(props);
+    this.handleNotificationDismissal = this.handleNotificationDismissal.bind(
```

Comment:
can avoid .bind and use property assignment that will have 'this' in scope

handleNotificationDismissal = ()=>{}

Reply:
Hmmm. Sure will do.

I got informed [from this](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md). Don't whats the right thing to do now. But I think having a consistent style is more important.

Reply:
The link is referring to having .bind / arrow functions inside JSX props or anywhere in the render() method, which will indeed create a new function everytime render() is called.
What you did (and also what I offered) are essentially the same in that they will not create a new function for each render() call since they are defined at the class level. It’s just more readable in my opinion.